internal/cloud/realm/dependencies.go

```diff
@@ -0,0 +1,87 @@
+package realm
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"mime"
+	"mime/multipart"
+	"net/http"
+	"os"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+const (
+	dependenciesPathPattern        = appPathPattern + "/dependencies"
+	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
+
+	paramFile = "file"
+)
+
+func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
```

Comment:
Is the linter not enforcing comments on exported funcs?

Reply:
the linter does enforce that.  ive noticed this in the past, but some exported funcs seem to not require a docstring/comment.  for instance ,`String` is definitely one.  i think for this case, there's something about implementing an interface method defined in the same package that you don't need the comment?